### ISSUE TITLE
Possible bug

### ISSUE THREAD

**Issue by zgelici** [Torann/laravel-geoip#142]
Hi,

I get an issue with a part of code. i did some checks.

vendor\torann\geoip\src\Services\IPApi.php

Line 74 gives notice:

`Notice: Trying to get property 'status' of non-object`

current code:
```
if ($json->status !== 'success') {
            throw new Exception('Request failed (' . $json->message . ')');
        }
```

i think this will solve it:

```
if ($json && $json->status !== 'success') {
            throw new Exception('Request failed (' . $json->message . ')');
        }
```

I get the issue when i call this:

`geoip()->getLocation($ip)['country']`

**Comment by jadjoubran** [comment]
I'm also getting that sometimes

**Comment by lamalamaMark** [comment]
We experience this error also frequently. @zgelici can you make a pull request containing your fix?

**Comment by zgelici** [comment]
> We experience this error also frequently. @zgelici can you make a pull request containing your fix?

I did it

**Comment by lamalamaMark** [comment]
I think the solution needs to be worked out further because the code after this condition also relies on the content of the $json variable. Did you look at that part?

**Comment by zgelici** [comment]
@lamalamaMark 

You are right, the other part need to be also inside $json check, but than its possible that the api or something gives empty results. What effect it will have.

**Comment by zgelici** [comment]
@lamalamaMark can you check now the new pull request

its possible that requests or something fails, so i did multi tries. if not it give empty. i didnt test it and its late here. tomorrow evening i will check my code again. but i already did push.

**Comment by paulocastellano** [comment]
I have same problem, sentry always alarm it.

**Comment by JexPY** [comment]
Same situation with sentry, but this is happening after update from Laravel 5.8 to 6.0

**Comment by huesoamz** [comment]
@lamalamaMark the fix of @zgelici its very useful why dont accept that PR?

**Comment by gutitrombotto** [comment]
Hi guys!

I have te same problem. 

Did you find the solution?

**Comment by wlepinski** [comment]
I just had the same problem today.
And I think it is related to the rate limiters defined [here](https://ip-api.com/docs/api:json).

```
Usage limits
This endpoint is limited to 45 requests per minute from an IP address.

If you go over the limit your requests will be throttled (HTTP 429) until your rate limit window is reset. If you constantly go over the limit your IP address will be banned for 1 hour.

The returned HTTP header X-Rl contains the number of requests remaining in the current rate limit window. X-Ttl contains the seconds until the limit is reset.
Your implementation should always check the value of the X-Rl header, and if its is 0 you must not send any more requests for the duration of X-Ttl in seconds.

We do not allow commercial use of this endpoint. Please see our pro service for SSL access, unlimited queries and commercial support.
```

While the code solves the problem with the non-object error it does not solve the problem of going above the limit on a non-paid environment.

**Comment by hosamalzagh** [comment]
same error
